core/src/test/java/org/infinispan/commons/util/concurrent/jdk8backported/BoundedEquivalentConcurrentHashMapV8BaseTest.java

```diff
@@ -32,6 +9,13 @@
 import org.infinispan.util.logging.LogFactory;
 import org.testng.annotations.Test;
 
+import java.util.Arrays;
+import java.util.Map;
```

Comment:
I reimported IDE settings just to make sure if this rearangement of imports is legal. According to my IntelliJ this is how it should look like...